data-serving/reusable-data-service/data_service/controller/case_controller.py

```diff
@@ -55,10 +55,12 @@ def list_cases(self, page: int = None, limit: int = None, filter: str = None):
         if limit <= 0:
             raise PreconditionUnsatisfiedError("limit must be >0")
         predicate = CaseController.parse_filter(filter)
+        print(f"predicate: {predicate}")
```

Comment:
`logging.debug`?

Reply:
Oops. Not even that: there's no logging anywhere in this yet, and I'll set that up universally later (ideally when I've worked out how to decorate the controller functions)